test/sagemaker_tests/pytorch/training/resources/distributed_operations.py

```diff
@@ -106,10 +106,10 @@ def _all_reduce(rank, rows, columns):
 def _reduce(rank, rows, columns):
     dest = 0
     tensor = _get_tensor(rank, rows, columns)
-    logger.debug("Rank: {},\nTensor BEFORE reduce: {}".format(rank, tensor))
+    # logger.debug("Rank: {},\nTensor BEFORE reduce: {}".format(rank, tensor))
```

Comment:
NIT: Can we remove these lines instead of commenting them out?

Reply:
I think the idea was to keep them for future use, if needed, where the debug logs for these parts can be enabled if more information is needed when debugging.

Reply:
Uncommented these lines, and running tests in [afcaccf](https://github.com/aws/deep-learning-containers/pull/2899/commits/afcaccf98f024cf3e7db8fd37f80eb8da7555e4f)